Aikido.Zen.Test/StatsTests.cs

```diff
@@ -435,9 +437,13 @@ public async Task ConcurrentOperations_ProduceConsistentResults()
                                 break;
                             case 2: // OnDetectedAttack (Global)
                                 bool blocked = random.Next(2) == 0;
-                                stats.OnDetectedAttack(blocked);
+                                stats.OnDetectedAttack();
                                 Interlocked.Increment(ref expectedGlobalAttacks);
-                                if (blocked) Interlocked.Increment(ref expectedGlobalBlocked);
+                                if (blocked)
+                                {
+                                    stats.OnBlockedAttack();
```

Comment:
not really part of the refactor, can we add the blocked boolean back to the OnDetectedAttack?